packages/smooth_app/lib/generic_lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -35,12 +35,46 @@ class SmoothAlertDialog extends StatelessWidget {
       shape: const RoundedRectangleBorder(
         borderRadius: ROUNDED_BORDER_RADIUS,
       ),
-      content: Column(
-        mainAxisSize: MainAxisSize.min,
-        children: <Widget>[
-          _buildTitle(context),
-          SizedBox(height: height, child: body),
-        ],
+      content: ConstrainedBox(
+        constraints: BoxConstraints(maxHeight: height ?? double.infinity),
```

Comment:
Also could you please rename the argument to SmoothAlertDialog from `height` to `maxHeight`